content/html/concepts/tags/terms/audio/audio.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '<audio>'
+Description: 'Represents an interface for adding audio content to the page.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Represents an interface for adding audio content to the page.
+
+HTML supports the following audio formats:
+
+- MP3
+- WAV
+- OGG
+
+## Syntax
+
+```html
+<audio controls>Source tags with different audio formats go here</audio>
+```
+
+The `<audio>` tag may contain one or more `<source>` tags with different audio formats. The browser will run the format most suitable for the browser.
+
+The `<audio>` tag may include `<source>` tags and fallback text. It may also use attributes to convey intention, some common ones are below:
+
+- `autoplay`: if this attribute is present the audio will play automatically.
+- `controls`: if this attribute is present the audio will have an overlay allowing the user to control the audio.
+- `loop`: specifies that when the audio ends, it will begin again.
+- `src`: the URL for the audio that should be shown to the user.
```

Comment:
```suggestion
- Using `autoplay` will automatically play the audio.
- Using `controls` will have an overlay to allow the user to control the audio.
- Using `loop` will specify that when the audio ends, it will begin again.
- Setting `src` to the URL for the audio to be shown to the user.
```